File: 0011-container-with-most-water/brute-force/brute-force.cs
public class Solution
{
    public int MaxArea(int[] height)
    {
        int n = height.Length;
        int max_area = 0;

        for (int i = 0; i < n; i++)
        {
            for (int j = 0; j < n; j++)
            {
                int area = (j - i) * Math.Min(height[i], height[j]); // Area = width * height
                max_area = Math.Max(area, max_area);
            }
        }

        return max_area;
    }
}

File: 0011-container-with-most-water/brute-force/brute-force.py
class Solution:
    def maxArea(self, height: List[int]) -> int:
        n = len(height)
        max_area = 0

        for i in range(n):
            for j in range(n):
                area = (j - i) * min(height[i], height[j]) # Area = width * height
                max_area = max(area, max_area)
        
        return max_area